src/main/kotlin/io/github/enyason/predictions/models/Input.kt

```diff
@@ -0,0 +1,5 @@
+package io.github.enyason.predictions.models
+
+data class Input(
+    var text: String? = null
```

Comment:
This `Input` object should map to what the user passed in.

Look at the input for this [prediction](https://replicate.com/stability-ai/sdxl?input=http&output=json), you will see that there is more than one property. It does not have `text` as a property either.

I suggest you use a map for this input type since we can't predict what each model is returning